### ISSUE TITLE
Faces change on the BBS

### ISSUE THREAD

**Issue by impaktor** [pioneerspacesim/pioneer#4810]
### Observed behaviour
Persistent characters, like FuelClub, and e.g "Carney's Warehouse" on Mars, get different faces each time you start a game.

### Expected behaviour
Same faces.

### Steps to reproduce
1. start new game on Mars
2. Go to Federal Fuel Club and/or "Carney's Warehouse", remember those horrible faces
3. restart the game, repeat.

I believe this migt have been introduced by #4775.

### code that seems to work:
Explicitly sending in the `rand` object to `Character.New()` in Goodstrader seems to solve it, but then I wonder: isn't that what the `faceseed` is for? ping @vakhoir 
```lua
local ad = {
	station  = station,
	flavour  = flavour,
	slogan   = slogan,
	ispolice = ispolice,
	faceseed = r:Integer(),
	trader   = Character.New({title = flavour, armour=false}, r),
}
```

**Comment by impaktor** [comment]
I think I've understood it.

First, fuelclub isn't coded to have persistent faces, so that one isn't a bug, but BreakdownServicing, and DonateToCranks are. The `faceseed` set in goodstrader is never used, but can be passed in as parameter, but then it's called `seed`.

I think I can fix this shortly.

**Comment by vakhoir** [comment]
Go ahead if you want to fix it, but just FYI I'll be going over all the mission modules while working on the Missions View. I didn't want to change them too much for the Bulletin Board, because these 2 views are interdependant. It might be a good idea to open a catch-all ticket for anything Missions/BBS related that comes up. Or I'll open a WIP PR and it can be tagged in any issue related to either Missions or BB.

**Comment by impaktor** [comment]
Sounds like a good idea, however, I'd like to fix this asap, as #4796 makes a case for being able to recognize the honest from the fake goodstrader, which currently is true only by shop-title, not by face.
